<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>FACTORIAL</string>
    </property>
    <widget class="QLineEdit" name="factInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="factCompute">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>COMPUTE</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="factProgress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>451</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="factPause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PAUSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="factCancel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RESULT</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="factResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>451</width>
       <height>271</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>471</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>PRIME NUMBERS</string>
    </property>
    <widget class="QLineEdit" name="primeFromInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="placeholderText">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="primeCompute">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>COMPUTE</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="primeProgress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>451</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="primePause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PAUSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="primeCancel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RESULT</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="primeResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>451</width>
       <height>271</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="primeToInput">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="placeholderText">
      <string>999</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
